RobertiOS/CardFortress
# Edit on main list and Remember me on log-in screen

## What problem am I fixing as part of this changes?
- Implementing microapps architecture using swift packages.
- Edit credit cards, and remember me option on log-in screen

## Screenshots:


## Steps to test edit credit cards:
- Tap on a card on the list.
- Edit the card.
- Tap on Save

## Steps to test "remenber me":
- enter your user name and password.
- check the checkbox on the login screen.
- tap login.
- logout and exit the app
- open the app again, the credentials should be there.
- tap login, you should be able to login without entering your username/ password
